model/note/note.go

```diff
@@ -239,6 +239,10 @@ func UpdateTitle(inst *instance.Instance, file *vfs.FileDoc, title string) error
 		return nil
 	}
 
+	event := Event{"title": title, "doctype": consts.NotesDocuments}
```

Comment:
This could go into a method in the `event.go` file like `PutTelepointer`. This way we'd have all possible realtime events for notes in the same place.